packages/core/src/components/TextArea/TextArea.types.ts

```diff
@@ -50,4 +50,12 @@ export interface TextAreaProps extends TextAreaNativeInputProps, VibeComponentPr
    * Placeholder text to display when the textarea is empty.
    */
   placeholder?: string;
+  /**
+   * The allowed number of characters.
+   */
+  maxLength?: number;
```

Comment:
Maybe my Storybook is weird, but I assume I need to define these types here for storybook to pick it up